SCClassLibrary/Common/Collections/extLazy.sc

```diff
@@ -0,0 +1,7 @@
++ LazyEnvir{
+	put { arg key, obj;
+		if(obj.isFunction){obj=obj.inEnvir(this.envir)};
```

Comment:
I think it would be safer:

`if(obj.isFunction) { obj = obj.inEnvir(this) };`

This is important when a key is not yet filled, e.g. `~x = { ~y + 1 };` but `~y` is nil.

(btw it's also better to keep the formatting style of the rest of the file, so I added spaces).